tests/lib.rs

```diff
@@ -403,6 +405,7 @@ mod test {
     }
 
     #[tokio::test]
+    #[should_panic]
```

Comment:
I wasn't sure what to do about this test so I marked it as `#[should_panic]`. It appears to test something that is no longer valid with cedar-policy 3.0.0, but I didn't feel comfortable removing it altogether without confirmation first.

Reply:
Let's keep the test for now. Can you please rename it to `authorize_with_duplicated_input_entities_should_panic`?

Reply:
Done!